### DISCUSSION TITLE
change icon color when selected

### DISCUSSION THREAD

**Discussion by JumpyLionnn** [davatorium/rofi#2014] [Q&A] [answered]
is there a way to change an `element-icon`'s color when its selected?
i am using a png file as my icon in a custom script (`\0icon\x1ficon.png\n`). is there a way to tint it or perhaps change the image when its selected?
i tried looking at the `icon-name` element but it doesnt work using the script

thanks in advance

**Comment by DaveDavenport** [comment]
No, this is not supported. You cannot change the color/tint of an image.

**Reply by DaveDavenport** [reply, marked as answer]
you can render text and text icons with a color as icon, 

From manpage:

```
echo -en "testing\0icon\x1f<span color='red'>⏻</span>" | rofi -dmenu
```

**Reply by ClairDePrince** [reply]
> you can render text and text icons with a color as icon,
> 
> From manpage:
> 
> ```
> echo -en "testing\0icon\x1f<span color='red'>⏻</span>" | rofi -dmenu
> ```

Can we change the icon color depending on if the entry is selected or not?

**Reply by DaveDavenport** [reply]
The 'tint' option should allow you to do this, this will be in the upcoming release.

**Reply by DaveDavenport** [reply]
https://github.com/davatorium/rofi/pull/2193

**Reply by ClairDePrince** [reply]
Thanks! So for the moment, if I have:

`echo -en "testing\0icon\x1f<span>⏻</span>\n""testing (not selected)\0icon\x1f<span>⏻</span>" | rofi -dmenu -show-icons`

which gives:
<img width="697" height="239" alt="Screenshot_16-06-2026_22-36-04" src="https://github.com/user-attachments/assets/41d9a87a-ffae-4de2-8806-a1369a737144" />

It's not currently possible to make sure the icon "⏻" have the same color as "testing" when it's selected? In this example, I want to make it white when it's selected and black when it's not
